showcase/src/assets/translations/en.json

```diff
@@ -6,6 +6,16 @@
         "CLASSIC-COMPACT": "Action Bar Classic Compact",
         "ALT": "Action Bar Alternative"
       },
+      "HEADER": {
+        "TITLE": "Stark App Header Guidelines",
+        "INTRO": "Stark provide you usefull classes to implement your application header. You just need to include the header theme in your SCSS",
+        "COPY_PASTE": "Then you can copy/paste this HTML into your code",
+        "STARK_APP_HEADER": "The <b>stark-app-header</b> provide a background to the header",
+        "STARK_APP_BAR": "The <b>stark-app-bar</b> contains three containers <b>stark-app-bar-content-left</b>, <b>stark-content-center</b>, <b>stark-content-right</b> which split the header in three parts. All the direct children of those 3 containers will be aligned in a column.",
+        "STARK_APP_BAR_CONTENT_LEFT": "<b>stark-app-bar-content-left</b> contains 2 containers <b>stark-actions</b> for the navigation actions and <b>stark-page-title</b> which displays the current page title.",
+        "STARK_APP_BAR_CONTENT_CENTER": "<b>stark-content-center</b> contains the app logo",
+        "STARK_APP_BAR_CONTENT_RIGHT": "<b>stark-content-right</b> contains 3 containers, the first one for the App Data component and the logout button, the second for language selection and <b>stark-app-bar-content-right-actions-alt</b> which contains some alternative actions."
```

Comment:
"stark-content-right" => "stark-app-bar-content-right"

The same for the FR and NL translations